.travis.yml

```diff
@@ -38,5 +38,6 @@ branches:
     only:
       - master
       - ora-staff-grading
+      - ned/django-upgrade-1.8
```

Comment:
@muhammad-ammar is this something we want to stay in the file? It seems strange to have a "ned" branch permanently part of the configuration.